lib/types/cql-utils.js

```diff
@@ -7,17 +7,72 @@ const TimeUuid = require("./time-uuid");
 const Uuid = require("./uuid");
 const Duration = require("./duration");
 
+/**
+ * Ensures the value isn't one of many ways to express null value
+ * @param {string} message Error message if value is "null"
+ * @throws {TypeError} Throws an error if the value is "null"
+ */
+function ensureValue(val, message) {
+    if (val === null || typeof val === "undefined" || val === types.unset)
+        throw new TypeError(message);
+}
+
+/**
+ * Wraps each of the elements into given subtype
+ * @param {Array} values
+ * @param {rust.ComplexType} subtype
+ * @returns {Array<rust.QueryParameterWrapper>}
+ */
+function encodeList(values, subtype) {
```

Comment:
:pick: In ScyllaDB code, List and Set both belong to the group called `Listlike` - you can use the name here, too.